pkg/skaffold/runner/build.go

```diff
@@ -29,6 +29,22 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func (r *SkaffoldRunner) buildAndTest(ctx context.Context, out io.Writer, artifacts []*latest.Artifact) error {
```

Comment:
A bit confusing to have `BuildAndTest` and `buildAndTest`?

Reply:
Yes, I know. I tried hard to simplify but need to try harder